### PR TITLE
Adopt a new way to write the log labels 

### PR DESCRIPTION
**Pull request by DevUt**

### CODE REVIEW

File: C/kmsg_log.c [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #include <unistd.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
```
Comment (nailyk-fr) [inline C/kmsg_log.c:3]: Should be done into 461266350c80271865efb1ef616892d9cfbe25a1

File: C/kmsg_log.c [hunk 2]
```diff
@@ -10,45 +10,45 @@
...
+  currently if it cant open kmsg it logs it
+  in the /cache/lible.log
+  */
 int klog(char *msg)
 {
 
  char *buf = malloc(256);
  int ret = -1;
 
-  // Write to kmsg if possibe.
```
Comment (nailyk-fr) [inline C/kmsg_log.c:19]: Why removing comments?

File: C/kmsg_log.c [hunk 3]
```diff
@@ -58,50 +58,54 @@ int klog(char *msg)
...
-	{
-    char *warn = malloc(256);
-     strcpy(warn ,"Warning!: ");
-     strcat(warn ,warning);
-  	int ret = klog(warn) ;
-      return(ret);
-    free(warn);
-	}
+{
+  char *warn = malloc(256);
```
Comment (nailyk-fr) [inline C/kmsg_log.c:61]: What happens to the indent? 
(on every commits)

File: C/kmsg_log.c [hunk 4]
```diff
@@ -14,14 +14,14 @@
 int klog(char *msg)
 {
 
- char *buf = malloc(sizeof(char) * 256);
+ char *buf = malloc(256);
```
Comment (nailyk-fr) [inline C/kmsg_log.c:24]: ok: https://stackoverflow.com/questions/419022/char-x256-vs-char-malloc256sizeofchar/419029#419029
btw char buf[256]; could be better.

File: C/kmsg_log.c [hunk 5]
```diff
@@ -52,32 +52,56 @@ int klog(char *msg)
...
+  to the real message and sends it to klog
+  for real "printing" to kmsg (IF LOG_LOC not defined or used)
+*/
+int logwarn(char *warning)
 	{
-		char *warn ="Warning! : "  ;
-		strcat(*warn,*warning);
-		int ret = klog(*warn);
-		return(ret);
+    char *warn = malloc(256);
```
Comment (nailyk-fr) [inline C/kmsg_log.c]: Ident problem here, as in every changes.

File: C/kmsg_log.c [hunk 6]
```diff
@@ -1,3 +1,4 @@
+#include <unistd.h>
```
Comment (nailyk-fr) [inline C/kmsg_log.c:1]: Ok on this commit.

**Review by nailyk-fr — CHANGES_REQUESTED** [review]
Good catches but need a bit of rework.

File: C/kmsg_log.c (hunk 3 above)
Comment (DevUt) [reply C/kmsg_log.c:61]: On the github webview  the original code was written but this code is written in Atom and is presented it wisely and making me push it so I didn't get that difference

File: C/kmsg_log.c (hunk 2 above)
Comment (DevUt) [reply C/kmsg_log.c:19]: A briefing about what a function does is enough. We dont need to Comment everything , our code should be able to explain whats happening

Comment when you sense reader wont know whats happening

Comment (nailyk-fr) [reply C/kmsg_log.c:19]: Every lines are obscure for me :p 
So ok for this too.

**Pull request closed without merging**